### PR TITLE
Support for topolo tree experiment measurements and latency

### PR DESCRIPTION
**Pull request by conor-23**

### CODE REVIEW

File: hydroflow_lang/src/graph/ops/fold.rs [hunk 1]
```diff
@@ -110,7 +110,7 @@ pub const FOLD: OperatorConstraints = OperatorConstraints {
                     };
                 },
                 quote_spanned! {op_span=>
-                    #context.schedule_subgraph(#context.current_subgraph(), false);
+                    // #context.schedule_subgraph(#context.current_subgraph(), false);
```
Comment (shadaj) [inline hydroflow_lang/src/graph/ops/fold.rs:113]: hmm why is this here

Comment (zzlk) [reply hydroflow_lang/src/graph/ops/fold.rs:113]: I think we added this to make fold stop spinning but this is no longer necessary.
